CRToast/CRToast.m

```diff
@@ -1049,8 +1067,18 @@ - (void)layoutSubviews {
                                       imageSize.height == 0 ?
                                       0 :
                                       CGRectGetHeight(contentFrame));
+
+    
+    if (self.toast.contentViewConfigurationBlock) {
+        self.contentView.frame = CGRectMake(CGRectGetWidth(contentFrame)-CGRectGetHeight(contentFrame)-kCRStatusBarViewNoImageRightContentInset,
+                                            statusBarYOffset,
+                                            CGRectGetHeight(contentFrame),
+                                            CGRectGetHeight(contentFrame));
```

Comment:
Why are the dimensions done this way? This seems make the view square and off the right. It'd be nice, at least in my use case, if it filled the whole toast.